worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -125,10 +132,16 @@ class PushDataHandler extends BaseMessageHandler with Logging {
     val mode = PartitionLocation.getMode(pushData.mode)
     val body = pushData.body.asInstanceOf[NettyManagedBuffer].getBuf
     val isMaster = mode == PartitionLocation.Mode.MASTER
+    val batchId = if (isMaster) pushState.nextBatchId() else -1
```

Comment:
I think should use the batchId inside the data

Reply:
> I think should use the batchId inside the data

Cost too much and no meaning for replicate data.

Reply:
It's a little strange that we use a different batchId than it should be.